package.json

```
@@ -13,7 +13,7 @@
     "@ldapjs/asn1": "2.0.0",
     "@ldapjs/attribute": "1.0.0",
     "@ldapjs/change": "1.0.0",
-    "@ldapjs/controls": "2.0.0",
+    "@ldapjs/controls": "2.1.0",
```

Ugh, some tool I used removed the `^` qualifier. Would you mind adding it to all of the dependencies as part of this PR?

```suggestion
    "@ldapjs/controls": "^2.1.0",
```

---

Sure, of course